### DISCUSSION TITLE
BentoML is slow with numpy array requests

### DISCUSSION THREAD

**Discussion by viethoang303** [bentoml/BentoML#3760] [Q&A]
Hi author and everyone, 

I'm a newbie with bentoml and I just used it recently? And I have a question:
Is there any way to request numpy array in bentoml, i noticed that bentoml request and response time is very slow?

Please help me

**Comment by larme** [comment]
Hi, Hoang. What is the size of the numpy array? Could you give an example project to reproduce the problem.
